tools/lib/project2html.mjs

```diff
@@ -516,15 +554,19 @@ export async function convertProjectToHTML(project, cliParams) {
     slot: session.slot,
     meeting: session.meeting?.split(';').map(m => m.trim())
   }));
-  writeLine(2, `
-    <section id="yaml">
-      <h2>Data for Saving/Restoring Schedule</h2>
-      <pre id="data">`);
-  writeLine(0, YAML.stringify(yaml));
-  writeLine(3, `</pre>
-    </section>
-  </body>
-</html>`);
-
+  if (!reduce) {
+    writeLine(2, `
+      <section id="yaml">
+        <h2>Data for Saving/Restoring Schedule</h2>
+        <pre id="data">`);
+    writeLine(0, YAML.stringify(yaml));
+    writeLine(3, `</pre>
+      </section>`);
+  }
+  if (reduce) {
+     writeLine(2, `<!-- Seed: ${cliParams.seed} -->`);
+  }
```

Comment:
Trying to hide parameters in the HTML page?

Same as in previous comments, if that seems useful for the reduce case, it is a worst useless (but not harmful) for the non-reduced case, so no need to switch on "reduce" for that, let's just add it everywhere!

Not a big deal but I'm thinking that a cleaner way to embed the seed would be to add it as a `<meta>` line in the header instead (to be moved to the beginning of the function):

```js
writeLine(2, `<meta name="seed" content="${cliParams.seed}">`);
```